### PR TITLE
Moved IPPR to completed projects

### PR DESCRIPTION
**Pull request by lili2311**
@womenhackfornonprofits/whfnp
## Linked Jira/Github Issue Ticket

[Issue-98](https://github.com/womenhackfornonprofits/whfnp/issues/98)
## What are the changes included in this PR?
- Moved IPPR to Completed Projects
- Fixed spacing/formatting

### CODE REVIEW

File: src/projects.html [hunk 1]
```diff
@@ -1,87 +1,143 @@
...
-          </div>
-        </div>
-      </div>
-    </section>
-    <section>
-      <div class="content text-section">
+    </div>
+</section>
+<section>
+    <div class="content text-section">
```
Comment (Icicleta) [inline src/projects.html:136]: should maybe `text-section` class goes on `<section>` as you did on rest of `<section>'s`

File: src/projects.html [hunk 2]
```diff
@@ -1,87 +1,143 @@
...
+    <div class="content">
         <h1 class="title-heading">Our Projects</h1>
-      </div>
-    </section>
-    <section>
-      <div class="content">
-        <h2 class="section-heading">Completed Projects </h2>
+    </div>
+</section>
+<section>
```
Comment (Icicleta) [inline src/projects.html:10]: I wonder whether you will need a class on that section to later on style this section in particular.

Comment (lili2311) [inline src/projects.html:10]: the section colour is automatic by the section number in the css.

File: src/projects.html (hunk 1 above)
Comment (lili2311) [inline src/projects.html:136]: text-section class doesn't even exist anymore, I will raise a ticket to refactor this.

**Pull request merged**